### PR TITLE
Add script to delete couch phone numbers

### PR DESCRIPTION
**Pull request by gcapalbo**
@dannyroberts

### CODE REVIEW

File: corehq/apps/sms/management/commands/delete_messaging_couch_phone_numbers.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+from corehq.apps.sms.mixin import VerifiedNumber
+from corehq.apps.sms.models import PhoneNumber
+from dimagi.utils.couch.database import iter_docs_with_retry, iter_bulk_delete_with_doc_type_verification
```
Comment (dimagimon) [inline corehq/apps/sms/management/commands/delete_messaging_couch_phone_numbers.py]: F401 'iter_docs_with_retry' imported but unused

File: corehq/apps/sms/management/commands/delete_messaging_couch_phone_numbers.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
+from corehq.apps.sms.mixin import VerifiedNumber
+from corehq.apps.sms.models import PhoneNumber
```
Comment (dimagimon) [inline corehq/apps/sms/management/commands/delete_messaging_couch_phone_numbers.py]: F401 'PhoneNumber' imported but unused

**Comment by gcapalbo** [comment]
(cancelled build, no test coverage)

**Comment by czue** [comment]
:skull:

**Pull request merged**